### DISCUSSION TITLE
Question to job queuing

### DISCUSSION THREAD

**Discussion by Apollo3zehn** [semaphoreui/semaphore#1632] [Q&A] [answered]
I have a task than runs once per day. If that task takes longer than a single day, will Semaphore wait for the task to complete or start another instance of that task? I am asking because right now I am executing the task's command with `flock` to prevent it running in parallel.

Thanks!

**Comment by fiftin** [comment, marked as answer]
Hi @Apollo3zehn it will be added to queue.
